debug-and-error-logs.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Debug & Error Logs
+summary: Learn how to find and read CockroachDB error logs
+toc: false
+---
+
+If you need to [troubleshoot](troubleshooting-overview.html) issues with your cluster, you can check your node's logs, which includes details about certain node-level and range-level events, such as errors. For example, if CockroachDB crashes, it normally logs a stack trace to what caused the problem.
+
+Logs are generated only on a per-node basis, which means each node's logs detail only the commands it processes without visibility into the behavior of other nodes in the cluster. When troubleshooting, this means that you must identify the node where the problem occurred or get the logs from all nodes in your cluster.
```

Comment:
... each node's logs detail only  the internal activity of that node without ...